openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -133,12 +133,11 @@ def test_dry_run_ligand(benzene_system, toluene_system,
         assert sampler.is_periodic
 
 
-@pytest.mark.parametrize('method', ['repex', 'sams', 'independent'])
 def test_dry_run_complex(benzene_complex_system, toluene_complex_system,
```

Comment:
so adding this as a slow test does make CI go down to ~ couple of minutes (at least on Linux), however it does mean not having recorded coverage for the "protein" bits of the Protocol. It's ~ 1%, so I don't know if we care that much.

Reply:
pretty sure we can mark individual parameters as slow here, so you could keep repex as always on, then test the other two when slow tests are enabled.

Reply:
tbh the other two cases (sams and independent) don't cover anything that's not covered in the vacuum and solvent tests, so I'd advocate for removing them and keeping repex.

My main question is whether we want to take the 4 mins bullet for this one repex test (which will give us coverage for protein paths).

Reply:
depends on your viewpoint of coverage.  internally we know these are doing the same thing which is your point, but externally it looks like two different codepaths so we probably ought to test both.  I'd probably wrap all three in a slow call?